artpaint/controls/CMYColorControl.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2022, Dale Cieslak
+ * Distributed under the terms of the MIT License
+ *
+ * Authors:
+ *      Dale Cieslak <dcieslak@yahoo.com>
+ *
+ */
+
+#include "CMYColorControl.h"
+
+
+#include "UtilityClasses.h"
+
+
+#include <Catalog.h>
+
+
+#undef B_TRANSLATION_CONTEXT
+#define B_TRANSLATION_CONTEXT "ColorSliders"
+
+
+CMYColorControl::CMYColorControl(rgb_color c)
+ 	: MultichannelColorControl(c, "C", "M", "Y", "A")
+{
+	slider1->SetResolution(1);
```

Comment:
To make translations easier, how about a B_TRANSLATE_COMMENT("Cyan", "For color model CMY") etc. for all tool tips?
While it's quite obvious for the standard RGB and CMY, HSV and CIELAB may be more challenging when you stumble over a term in isolation.
Generally, it's not a bad idea to use B_TRANSLATE_COMMENT more often to give hints to the translators.

Reply:
I added comments - I wasn't aware of B_TRANSLATE_COMMENT but it's a great idea especially for ambiguous terms like "value"